aesara/graph/basic.py

```diff
@@ -1543,13 +1543,15 @@ def equal_computations(xs, ys, in_xs=None, in_ys=None):
         if x.owner:  # Check above tell that y.owner eval to True too.
             if x.owner.outputs.index(x) != y.owner.outputs.index(y):
                 return False
+        # TODO: Should we use `Type.is_compatible`?
```

Comment:
Can you explain to me why we aren't able to use `Type.is_compatible` over here (and similar such places) ?

Reply:
See [the comment above](https://github.com/aesara-devs/aesara/pull/711#discussion_r776805017).